lib/scripts/update_agent_config.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+
+# Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2.0 License.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2022-Present Datadog, Inc.
+
+DATADOG_DIR="${DATADOG_DIR:-/home/vcap/app/.datadog}"
+SUPPRESS_DD_AGENT_OUTPUT="${SUPPRESS_DD_AGENT_OUTPUT:-true}"
+export DD_TAGS=$(LEGACY_TAGS_FORMAT=true python $DATADOG_DIR/scripts/get_tags.py)
+
+datadog_tags=$(python $DATADOG_DIR/scripts/get_tags.py node-agent-tags)
```

Comment:
```suggestion
datadog_tags=$(python $DATADOG_DIR/scripts/get_tags.py node-agent-tags)
export DD_TAGS=""
```

Reply:
Added it on line 41 so the logs config script also uses it. I don't think this will ever be needed though now that we unset it in run-datadog.sh